applications/nrf_desktop/src/modules/led_stream.c

```diff
@@ -76,7 +84,7 @@ static bool queue_data(const uint8_t *data, const size_t size, struct led *led)
 
 	/* Fill only leds available in the system */
 	memcpy(led->steps_queue[led->rx_idx].color.c, &data[pos],
```

Comment:
Values must be scaled. CAF LEDs accept values in range 0-100, we may receive values in range 0-255 from config channel.

Reply:
Also we could consider upgrading the algorithm used to generate or forward colors if only one channel is supported.

In that case averaging 3 values is not the best idea (for random color we reduce entropy averaging 3 subsequent random values and for music LED stream averaging actually causes LED to ignore hue modifications that are used to indicate changes in music).

Reply:
added conversion
I prefer to not touch the led_stream at the moment. it is pandora box

Reply:
We could simply check how many channels are supported by a given LED and in case there is only one channel we could send r, g and b values set to received r channel value. In that way, we would keep the previous behavior. 
With current implementation it looks like music LED stream is broken for single channel LED - e.g. on DK (I am not sure if anyone uses the LED stream in that configuration).

Reply:
I could add this in separate PR

Reply:
could add that but so far I don't think anybody uses led stream. it is a nice demo feature but unless we get any information about its usability I would not invest in this area